frontend/src/components/AssetPrice.vue

```diff
@@ -43,28 +43,31 @@
 </template>
 
 <script lang="ts">
-import {computed, defineComponent} from 'vue'
+import {computed, defineComponent, ref} from 'vue'
 import {Money} from 'ts-money'
 
 export default defineComponent({
   props: {
     currentPrice: {
-      type     : Object as Money,
+      type     : Money,
       required : true
     },
     previousPrice: {
-      type    : Object as Money,
+      type    : Money,
       default : null
     },
   },
   setup (props) {
-    const change: Money = computed(() => props.currentPrice.subtract(props.previousPrice))
+    const price = ref(props.currentPrice)
```

Comment:
I thought props dont need to be wrapped in refs

Reply:
No clues, I can check if it can get removed

Reply:
@mkitzmann You're right, but be cautious if that prop is optional.

https://v3.vuejs.org/guide/composition-api-setup.html#props

> If title is an optional prop, it could be missing from props. In that case, toRefs won't create a ref for title. Instead you'd need to use toRef:

```js
setup(props) {
  const title = toRef(props, 'title')

  console.log(title.value)
}
```